src/components/HelpDrawer/HelpContent/PreferencesHelpComponent.tsx

```diff
@@ -27,6 +27,7 @@ export class PreferencesHelpComponent extends React.Component {
                     <li>WCS matching on append: trigger WCS matching automatically for newly appended images</li>
                     <li>Spectral matching: spectral convention to be used for spectral matching of image cubes</li>
                     <li>Transparent image background: when this is enabled, the exported png image will have a transparent background. When it is disabled (default), a white or a black background is added depending on the GUI theme.</li>
+                    <li>Save last used directory: when this is enabled, the file browser will start in the same folder the next session.</li>
```

Comment:
thank you for doing this! we always forget to add these and then @kswang1029 has to sift through all the changes :see_no_evil:

Reply:
I was lucky to catch this in the latest changes merged into the codebase! I hope I keep the good practice.